osl/namespace_linux.go

```diff
@@ -498,3 +514,79 @@ func (n *networkNamespace) Restore(ifsopt map[string][]IfaceOption, routes []*ty
 
 	return nil
 }
+
+// Checks whether IPv6 needs to be enabled/disabled on the loopback interface
+func (n *networkNamespace) checkLoV6() {
+	var (
+		loV6   = false
+		action = "disable"
+	)
+
+	n.Lock()
+	for _, iface := range n.iFaces {
+		if iface.AddressIPv6() != nil {
+			loV6 = true
+			action = "enable"
+			break
+		}
+	}
+	n.Unlock()
+
+	if err := setIPv6(n.path, "lo", loV6); err != nil {
+		logrus.Warnf("Failed to %s IPv6 on loopback interface on network namespace %q: %v", action, n.path, err)
+	}
+}
+
+func reexecSetIPv6() {
+	runtime.LockOSThread()
+	defer runtime.UnlockOSThread()
+
+	if len(os.Args) < 3 {
+		logrus.Errorf("invalid number of arguments for %s", os.Args[0])
+		os.Exit(1)
+	}
+
+	f, err := os.OpenFile(os.Args[1], os.O_RDONLY, 0)
```

Comment:
Why can't  we use `netns.GetFromPath` api to do this instead of repeating this everywhere?